### DISCUSSION TITLE
Lost my student pack

### DISCUSSION THREAD

**Discussion by freiresf** [community/community#68567] [GitHub Education]
### Select Topic Area

Question

### Body

i lose access to my previous account but i am not able to change my student pack content, does any one knows who i could contact? No one is answering my ticket and i just cant use any student tools that i needed most at university

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @freiresf , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by freiresf** [reply]
hello! Thanks for your advice but where could i post? Little lost here, tried some contacts

**Reply by queenofcorgis** [reply]
For your student pack-related questions, please use the [GitHub Education category](https://github.com/orgs/community/discussions/categories/github-education),  however no need to repost this question as I've moved it to the category.
